src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -41,6 +41,21 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         Person personToDelete = lastShownList.get(targetIndex.getZeroBased());
+
+        /*
+        this part of code should only exist in javafx file so remove it so far
+        Show confirmation dialog
+        Alert alert = new Alert(Alert.AlertType.CONFIRMATION);
+        alert.setTitle("Delete Confirmation");
+        alert.setHeaderText("Are you sure you want to delete this person?");
+        alert.setContentText(String.format("Name: %s", personToDelete.getName()));
+
+        ButtonType result = alert.showAndWait().orElse(ButtonType.CANCEL);
+        if (result != ButtonType.OK) {
+            return new CommandResult("Deletion cancelled.");
+        }
+         */
+
```

Comment:
What does this mean by should only exist in javafx file?

Reply:
the gradlew test doesn't allow this tip command to be in delete command class I don't know how to solve this so commented this part of code so far, try to solve it the next time